neo/Network/P2P/Payloads/BlockBase.cs

```diff
@@ -35,11 +35,11 @@ public UInt256 Hash
 
         public virtual int Size =>
             sizeof(uint) +       //Version
-            PrevHash.Size +      //PrevHash
-            MerkleRoot.Size +    //MerkleRoot
```

Comment:
Why is this change required?

Reply:
Is not required, but maybe you need to know the size without these values. Is that case